tools/mac_setup.sh

```diff
@@ -28,8 +29,8 @@ brew "libarchive"
 brew "libusb"
 brew "libtool"
 brew "llvm"
+brew "curl"
 brew "openssl"
-brew "pyenv"
```

Comment:
why was this removed? I think it's preferable to install with brew in this case

Reply:
this version of `pyenv` oddly doesn't have the `update` command used in `update_requirements.sh` https://github.com/commaai/openpilot/blob/d5bdf1e951a2382b0df588b18183423d0ac26b56/update_requirements.sh#L18

Reply:
still want this to work on the version installed with brew

Reply:
back in brew